packages/autocomplete-css/update.js

```diff
@@ -0,0 +1,300 @@
+      if (summaryRaw[i].length > 1) {
+        return summaryRaw[i]
+          .replace(/\{\{\S+\("(\S+)"\)\}\}/g, '$1')
+          .replace(/\*/g, "")
+          .replace(/\`/g, "")
+          .replace(/\{/g, "")
+          .replace(/\}/g, "")
+          .replace(/\"/g, "")
+          .replace(/\[([A-Za-z0-9-_* ]+)\]\(\S+\)/g, '$1');
+      }
+    }
+  } else {
+    // A document doesn't yet exist. Let's return an empty value.
+    return "";
+  }
+}
+
+function getValuesOfProp(value, allValues) {
+  // value holds the value string of the values we expect
+  // allValues holds all of the values that apply to the spec
+  // Like mentioned above `value` = "value1 | value2 | <valueGroupName>"
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/Value_definition_syntax
+
+  if (!value || value.length < 0) {
+    return [];
+  }
+
+  let values = [];
+  let parser = new CSSParser(value);
+
+  let rawArrayValues = parser.parse();
+
+  for (const val of rawArrayValues) {
+    if (val.length > 1) {
+      // Since some values contain `||` some splits leave a zero length string
+      if (val.trim().startsWith("<") && val.trim().endsWith(">")) {
+        // This is a valueGroup lookup key
+        let valueGroup = parseValueGroup(val.trim(), allValues);
+        values = values.concat(valueGroup);
+      } else {
+        values.push(val.trim());
+      }
+    }
+  }
+
+  return values;
+
+}
+
+function parseValueGroup(valueGroupName, allValues) {
+  // Will lookup a valueGroup name within allValues and parse it
+
+  let resolvedValueGroupString;
+
+  // Now we can receive two kinds of Basic Data Types here
+  // - Non-Terminal Data Types: <'valueGroupName'> which will share the name of a property
+  // - And the standard that this was built to deal with.
+
+  if (valueGroupName.startsWith("<'") && valueGroupName.endsWith("'>")) {
+    // Non-Terminal Data Types
+    for (const spec in allValues) {
+      if (Array.isArray(allValues[spec].properties)) {
+        for (const prop of allValues[spec].properties) {
+          if (prop.name === valueGroupName.replace("<'", "").replace("'>", "")) {
+            resolvedValueGroupString = prop.value;
+            break;
+          }
+        }
+      }
+    }
+  } else {
+    // Standard handling
+    for (const spec in allValues) {
+      if (Array.isArray(allValues[spec].values)) {
+        for (const val of allValues[spec].values) {
+          if (val.name === valueGroupName) {
+            resolvedValueGroupString = val.value;
+            break;
+          }
+        }
+      }
+    }
+  }
+
+  return getValuesOfProp(resolvedValueGroupString);
+}
+
+async function getTagsHTML() {
+  // This will also use our dep of `mdn/content` to find all tags currently
+  // within their docs. By simply grabbing all folders of tag docs by their name
+
+  // Some of the page titles from MDN's docs don't accurately reflect what we
+  // would expect to apper. The object below is named after what the name of the
```

Comment:
```suggestion
  // would expect to appear. The object below is named after what the name of the
```
random typo heh. makes me think there's probably a typo-comment-linter out there, though of course there's probably going to be a lot of false positives...